tests/jobs/test_scheduler_job.py

```diff
@@ -3579,9 +3579,9 @@ def setUp(self) -> None:
             # One DAG with one task per DAG file
             (13, 1, 1),  # noqa
             # One DAG with five tasks per DAG  file
-            (21, 1, 5),  # noqa
+            (17, 1, 5),  # noqa
             # 10 DAGs with 10 tasks per DAG file
-            (77, 10, 10),  # noqa
+            (46, 10, 10),  # noqa
```

Comment:
Call `session.commit()` less often, resulting in fewer queries. This is a side-effect of this change rather than a goal.

Reply:
🚀